src/plugins/mentionAvatars/index.tsx

```diff
@@ -6,16 +6,25 @@
 
 import "./styles.css";
 
+import { definePluginSettings } from "@api/Settings";
 import ErrorBoundary from "@components/ErrorBoundary";
 import { Devs } from "@utils/constants";
-import definePlugin from "@utils/types";
+import definePlugin, { OptionType } from "@utils/types";
 import { SelectedGuildStore, useState } from "@webpack/common";
 import { User } from "discord-types/general";
 
+const settings = definePluginSettings({
+    hideAtSymbol: {
+        type: OptionType.BOOLEAN,
+        description: "Whether the the @-symbol should be hidden.",
+        default: false
+    }
+});
+
 export default definePlugin({
     name: "MentionAvatars",
     description: "Shows user avatars inside mentions",
-    authors: [Devs.Ven],
+    authors: [Devs.Ven, Devs.Luna],
```

Comment:
please don't add yourself for such minor changes

```suggestion
    authors: [Devs.Ven],
```